### DISCUSSION TITLE
Plotting Finished, Do I need to do anything to start farming?

### DISCUSSION THREAD

**Discussion by totomtornado** [swar/Swar-Chia-Plot-Manager#794] [Q&A]
I plan to have the GUI farming and use Swar-Chia-Plot-Manager for plotting.

I am unsure how plots are added to the farm but when using the GUI to plot, plots are automagically added to the farm after they finish.

If I use war-Chia-Plot-Manager, do I need to do anything to add plots to my farm or will the GUI just add them?

**Comment by Ron-ski** [comment]
In the GUI you need to click "Plots" on the left, there is then a green "+ ADD A PLOT" button to the right, next to that is three dots, click the three dots and you get the option to add a plot directory. The GUI will then monitor that directory and add the plots each time one appears. You need the GUI running all the time to farm. Good luck.

**Reply by totomtornado** [reply]
Yep, I noticed the GUI does in fact add the plots automatically as I was "watching" the folder before.  Cheers

**Comment by 7infinity7** [comment]
Hello
I did add the directory, but nothing happened, Would you please tell me what should i do because i think there is something wrong. 
I can not add the plots on GUI
Thank you

**Reply by Ron-ski** [reply]
You need to add the directory that has the plots in it, if they are in subfolders you need to add the subfolders. The plots also need to be valid and have your keys.

**Reply by 7infinity7** [reply]
you mean i need add my public key on the setting? what about pool key?

**Reply by 7infinity7** [reply]
Because i already added the directory but doesn't work.

**Reply by Ron-ski** [reply]
Where did you get your plots from? When you create plots it will use your keys, no need to add them. If the plots are valid, and you have the directory correct it should add them. Check if the plots are valid - use Google if you don't know how to, and be careful what you run and how. I'm at work so can't post the script to check them, but it may say at the bottom of the plot page if you have invalid plots.

**Reply by 7infinity7** [reply]
https://imgur.com/a/QSZDcMD
Please check this screenshot, plot are there but can not find
[https://imgur.com/a/QSZDcMD](url)

**Reply by Ron-ski** [reply]
Sorry I don't know the answer, see here it may help https://github.com/Chia-Network/chia-blockchain/issues/3522

Presuming you are on Windows you can check your plots are valid by opening a cmd window, pasting the next two lines one at a time. If you're not using the latest version, then I suggest you update to that, if you're using an earlier version and want to test before updating you'll need to change the 1.1.7 to your version number (use help - about to see version number)

CD %LOCALAPPDATA%\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon
.\chia plots check -g "G:\005"
